src/geo/transform.js

```diff
@@ -1355,11 +1361,9 @@ class Transform {
 
         this._constraining = true;
 
-        let minY = -90;
-        let maxY = 90;
-        let minX = -180;
-        let maxX = 180;
-        let sy, sx, x2, y2;
+        let minY = Infinity;
+        let maxY = -Infinity;
```

Comment:
As opposed to longitude, which wraps, these could never even possibly be outside the range [-90, 90], right, so that they can be initialized with those bounds?

```suggestion
        let maxY = -90;
```

Reply:
Not quite, these values are in pixel coordinates. (Those 90 and 180 values are a misleading!) I believe the correct range shouold be: `[0, this.worldSize]`. 

It makes no difference to functionality since the values are always modified before they are used, the only reason to initialize them is to prevent a Flow error. My first thought was that `[0, 0]` was the clearest way to show an unused value. But happy to change it to`[0, this.worldSize]` if you think that makes more sense!

Reply:
Ah, got it! Nah, infinities are great then. Yes, I assumed it was straight latitude/longitude, of course. LGTM!